src/window.py

```diff
@@ -101,6 +101,7 @@ class PortfolioWindow(Adw.ApplicationWindow):
     placeholder_box = Gtk.Template.Child()
     placeholder_inner_box = Gtk.Template.Child()
     menu_button = Gtk.Template.Child()
+    bookmark_button = Gtk.Template.Child()
```

Comment:
Design-wise, a few ideas:

* For the initial Places and Devices view, having a separate section for Bookmarks would make make sense.
* The Bookmark entries could have a remove button, similar to the Devices eject button, as it would save the extra indirection.
* The "bookmark current directory" button could be placed on the bottom action bar instead, since we need to save space for directory names.